lib/src/base-styles/_nx-lists.scss

```diff
@@ -70,12 +70,15 @@
   grid-area: list-text;
   margin: 0;
   padding-right: var(--nx-spacing-6x);
+  max-width: var(--nx-width-text-wrapping);
```

Comment:
Centering of empty text is broken.
![image](https://user-images.githubusercontent.com/3630091/124979735-6d672f00-e001-11eb-9f07-ac0ec9a7d4ff.png)

Reply:
> Centering of empty text is broken.

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/432/commits/f270a3be8f1a7edfe74f870805a9f4f9f2d72aae